pkg/kubelet/images/image_gc_manager_test.go

```diff
@@ -206,6 +206,105 @@ func TestDeleteUnusedImagesExemptSandboxImage(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestDeletePinnedImage(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
```

Comment:
no need to call `Finish` when passing a testing.T into NewController as of go1.14 (https://pkg.go.dev/github.com/golang/mock/gomock#Controller.Finish)

```suggestion

```